src/main/java/dev/plex/extras/command/ClownfishCommand.java

```diff
@@ -0,0 +1,72 @@
+package dev.plex.extras.command;
+
+import dev.plex.command.PlexCommand;
+import dev.plex.command.annotation.CommandParameters;
+import dev.plex.command.annotation.CommandPermissions;
+import dev.plex.extras.TFMExtras;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.minimessage.MiniMessage;
+import org.bukkit.Material;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Collections;
+import java.util.List;
+
+@CommandParameters(name = "clownfish", description = "Gives a player a clownfish capable of knocking people back")
+@CommandPermissions(permission = "plex.tfmextras.clownfish")
+public class ClownfishCommand extends PlexCommand
+{
+    @Override
+    protected Component execute(@NotNull CommandSender commandSender, @Nullable Player player, @NotNull String[] args)
+    {
+        if (args.length == 0)
+        {
+            ItemStack clownfish = new ItemStack(Material.TROPICAL_FISH);
+            ItemMeta meta = clownfish.getItemMeta();
+
+            meta.displayName(Component.text("Clownfish"));
+            clownfish.setItemMeta(meta);
+
+            player.getInventory().addItem(clownfish);
+            return MiniMessage.miniMessage().deserialize("<rainbow>blub blub... ><_>");
+        }
+        else if (args[0].equals("toggle"))
```

Comment:
There should be a way to prevent players from using the clownfish so that people who abuse it can have it taken away from them.